Makefile

```diff
@@ -341,7 +341,7 @@ commitlint: node_modules/.installed ## Run commitlint linter.
 	commitlint_from=$(COMMITLINT_FROM_REF); \
 	commitlint_to=$(COMMITLINT_TO_REF); \
 	if [ "$${commitlint_from}" == "" ]; then \
-		commitlint_from=$$(git remote show origin | grep 'HEAD branch' | awk '{print $$NF}'); \
+		commitlint_from=$$(git config --get init.defaultBranch || echo "main"); \
```

Comment:
The `init.defaultBranch` configuration setting is not likely to be set on the repository. This is a setting used to set the default branch for new repositories created by `git init`.

This does not solve the problem. The default branch that we care about is not that used by `git init` but the branch that is used as the `HEAD` branch by the git remote.